.github/workflows/curator-api-node.yml

```diff
@@ -33,3 +33,8 @@ jobs:
         working-directory: verification/curator-service/api
         env:
           CI: true
+      - name: Upload coverage
```

Comment:
Is this about test coverage? If so, could we name it as so?

Reply:
yup, done

Reply:
I find it confusing that the Codecov Report doesn't mention that it's test coverage anywhere. But I suppose there's nothing we can do about that.